packages/core/src/editor.ts

```diff
@@ -151,8 +152,12 @@ export class Editor {
                 if (!mark) throw new Error();
                 return (cur as Required<Mark>).keymap(mark);
             });
+        const historyKeymap = {
+            'Mod-z': undo,
+            'Shift-Mod-z': redo,
+        };
 
-        return [...nodesKeymap, ...marksKeymap].map((keys) => keymap(keys));
+        return [...nodesKeymap, ...marksKeymap, historyKeymap].map((keys) => keymap(keys));
```

Comment:
Move this line to 184 is better

Reply:
![image](https://user-images.githubusercontent.com/8351437/114639846-692cbd80-9d01-11eb-85e0-566ecc02cc4a.png)

You mean here?

Reply:
Yes